packages/protocol/graphics.ts

```diff
@@ -170,50 +64,20 @@ function onMouseEvents(events: MouseEvent[]) {
   }
 }
 
-export let hasAllPaintPoints = false;
-
-export const setHasAllPaintPoints = (newValue: boolean) => {
-  hasAllPaintPoints = newValue;
-};
 export const timeIsBeyondKnownPaints = (time: number) =>
-  !hasAllPaintPoints && gPaintPoints[gPaintPoints.length - 1].time < time;
+  fetchingPaints !== Status.Finished && lastReceivedPaintPoint().time < time;
 
-export function setupGraphics() {
+export function setupGraphics(replayClient: ReplayClientInterface) {
   ThreadFront.sessionWaiter.promise.then(async (sessionId: string) => {
-    let paintedGraphics = false;
-    const maybePaintGraphics = async () => {
-      if (!paintedGraphics) {
-        const { screen, mouse } = await getGraphicsAtTime(ThreadFront.currentTime, false, true);
-        if (screen && mouse) {
-          paintedGraphics = true;
-          paintGraphics(screen, mouse);
-        }
-      }
-    };
-
     const { client } = await import("./socket");
```

Comment:
Do we still need `client` here or could we just as easily make this `ReplayClient`? Or should this wait until we move those resources into suspense things?